ISSUES.md

```diff
@@ -0,0 +1,275 @@
+<!---
+Copyright 2020 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# How To Request Support
+
+This is an Open Source Project so please be mindful that like in any other project of this kind there is no obligation to answer all requests for help.
+
+This document's main purpose is to provide guidelines at how you can formulate your requests to increase your chances to be understood and to get supported.
```

Comment:
```suggestion
This document's main purpose is to provide guidelines at how you can formulate your requests to increase your chances to be understood and to get support.
```

I might be very wrong here, but isn't it "`get support`"?

Reply:
either way works. I merged your suggestion manually, thank you!